pkg/monitoring/alloy/templates/monitoring-config.yaml.template

```diff
@@ -89,6 +98,8 @@ verticalPodAutoscaler:
       - memory
       - cpu
       controlledValues: "RequestsAndLimits"
+      maxAllowed: 
+        cpu: 500m
       minAllowed:
```

Comment:
```suggestion
      minAllowed:
        cpu: 25m
        memory: 128Mi
```

Reply:
Arf, the suggestion didn't take into account the whole section

Reply:
What do you think about the current change? I don't think it makes sense to ask for less than 500 Mi for a prometheus anyway

Reply:
Yeah it makes sense :)